### PR TITLE
Updating README

### PR DESCRIPTION
**Pull request by apatterson-splunk**
Updating readme to add the option to send spans directly to ingest. Slack conversation linked below:

https://splunk.slack.com/archives/CQ17ZQJFP/p1630355869051600

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,7 +156,14 @@ template. For more information, see the [example](./examples/splunk-wrapper/READ
     OTEL_EXPORTER_JAEGER_ENDPOINT=http://127.0.0.1:9080/v1/trace
     SPLUNK_ACCESS_TOKEN="orgAccessToken"
    ``` 
-   
+   You can also use the `jaeger-thrift-splunk' exporter to send spans directly to the Splunk Observability Cloud backend. You can update accomplish this by updating OTEL_EXPORTER_JAEGER_ENDPOINT to the ingest URL.
```
Comment (mateuszrzeszutek) [inline README.md:159]:
```suggestion
   You can also use the `jaeger-thrift-splunk` exporter to send spans directly to the Splunk Observability Cloud backend. You can update accomplish this by updating `OTEL_EXPORTER_JAEGER_ENDPOINT` to the ingest URL.
```

**Comment by mateuszrzeszutek** [comment]
CC @theletterf

File: README.md (hunk 1 above)
Comment (theletterf) [inline README.md:159]:
```suggestion
   You can also use the `jaeger-thrift-splunk` exporter to send spans directly to the Splunk Observability Cloud backend. You can accomplish this by pointing `OTEL_EXPORTER_JAEGER_ENDPOINT` to the ingest URL.
```

**Review by theletterf — CHANGES_REQUESTED** [review]

**Comment by kubawach** [comment]
@mateuszrzeszutek @apatterson-splunk per splunk's doc (https://docs.signalfx.com/en/latest/apm/apm-getting-started/apm-supported-span-formats.html#apm-ingest-endpoint-compatible-span-formats) both jaeger-thrift and otlp are supported on the ingest, so I think the doc change should say that the user can select either exporter but has to set token and endpoint. WDYT?

**Comment by mateuszrzeszutek** [comment]
Ingest only supports HTTP+protobuf OTLP -- I'm not sure that the SDK supports that mode though, we should test that before we mention it in the docs.

**Review by kubawach — APPROVED** [review]

**Pull request merged**